warehouse/models/intermediate/gtfs/int_gtfs_schedule__daily_scheduled_service_index.sql

```diff
@@ -0,0 +1,83 @@
+{{ config(materialized='table') }}
+
+WITH dim_calendar_dates AS (
+    SELECT *
+    FROM {{ ref('dim_calendar_dates') }}
+),
+
+int_gtfs_schedule__long_calendar AS (
+    SELECT *
+    FROM {{ ref('int_gtfs_schedule__long_calendar') }}
+),
+
+fct_daily_schedule_feeds AS (
+    SELECT
+        *,
+        EXTRACT(DAYOFWEEK FROM date) AS day_num
+    FROM {{ ref('fct_daily_schedule_feeds') }}
+),
+
+boolean_calendar_dates AS (
+    SELECT
+        date,
+        feed_key,
+        service_id,
+        CASE
+            WHEN exception_type = 1 THEN TRUE
+            WHEN exception_type = 2 THEN FALSE
+        END AS service_indicator
```

Comment:
are there exception types other than 1 and 2? I think `exception_type = 1 AS service_indicator` would be equivalent.

also nit: if it's a boolean could we prefix with a verb? for example `has_service` or something

Reply:
No other exception types are valid. https://gtfs.org/schedule/reference/#calendar_datestxt. But it's possible there could be an invalid value, not sure we'd want to just lump that with `exception_type = 2`. Current `not_null` test would catch this.

Reply:
https://github.com/cal-itp/data-infra/pull/1936/commits/20d7da2a4f6d0f88665e7edb417d7e096117a8fd